build.xml

```diff
@@ -120,61 +77,134 @@
+            <arg path="${path.packages}/*/tests"/>
+            <arg path="${path.packages}/*/*.md"/>
             <arg path="${path.utils}/*/src"/>
             <arg path="${path.utils}/*/tests"/>
+            <arg path="${project.basedir}/*.md"/>
+            <arg path="${project.basedir}/docs"/>
         </exec>
+    </target>
+
+    <target name="ecs-fix-diff" description="Checks and fixes automatically fixable coding standards in all files in the whole monorepo by PHP easy coding standards.">
         <exec executable="${path.ecs.executable}" logoutput="true" passthru="true" checkreturn="true">
             <arg value="check"/>
+            <arg value="--fix"/>
+            <arg path="${path.src}"/>
+            <arg path="${path.tests}"/>
+            <arg path="${path.root}/*.md"/>
+            <arg path="${path.root}/docs"/>
+            <arg path="${path.packages}/*/src"/>
+            <arg path="${path.packages}/*/tests"/>
+            <arg path="${path.packages}/*/*.md"/>
+            <arg path="${path.utils}/*/src"/>
+            <arg path="${path.utils}/*/tests"/>
+            <arg path="${project.basedir}/*.md"/>
+            <arg path="${project.basedir}/docs"/>
+        </exec>
+    </target>
+
+    <target name="phplint" description="Checks syntax of PHP files in the whole monorepo.">
+        <exec executable="${path.phplint.executable}" logoutput="true" passthru="true" checkreturn="true">
+            <arg path="${path.src}"/>
+            <arg path="${path.tests}"/>
+            <arg path="${path.packages}/*/src"/>
+            <arg path="${path.packages}/*/tests"/>
             <arg path="${path.utils}/*/src"/>
             <arg path="${path.utils}/*/tests"/>
         </exec>
+    </target>
+
+    <target name="phplint-diff" hidden="true">
+        <echo level="info" message="Checking only changed files via 'phplint-diff' not supported in monorepo. Running 'phplint' instead..."/>
+        <phingcall target="phplint"/>
+    </target>
+
+    <target name="twig-lint" description="Checks syntax of Twig templates in the whole monorepo.">
+        <phingcall target="shopsys_framework.twig-lint"/>
+        <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
+            <arg value="${path.bin-console}"/>
+            <arg value="lint:twig"/>
+            <arg value="${path.packages}"/>
+        </exec>
+        <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
+            <arg value="${path.bin-console}"/>
+            <arg value="lint:twig"/>
+            <arg value="${path.utils}"/>
+        </exec>
+    </target>
+
+    <target name="twig-lint-diff" hidden="true">
+        <echo level="info" message="Checking only changed files via 'twig-lint-diff' not supported in monorepo. Running 'twig-lint' instead..."/>
+        <phingcall target="twig-lint"/>
+    </target>
+
+    <target name="yaml-lint" description="Checks syntax of Yaml files in the whole monorepo.">
```

Comment:
Unfortunately, the `lint:yaml` doesn't support multiple paths, wildcards or excluding directories (and there are problems in vendor and node_modules directories). This means I would have to call it one-by-one for all the root files.

I tried to make it work via `<foreach>` task, but it would be to complicated compared to the actual help it would provide.